nodejs/awsx/ec2/vpc.ts

```diff
@@ -652,10 +662,10 @@ export interface VpcArgs {
     subnets?: VpcSubnetArgs[];
 
     /**
-     * The maximum number of availability zones to use in the current region.  Defaults to `2` if
-     * unspecified.  Use `"all"` to use all the availability zones in the current region.
+     * The names of the availability zones to use in the current region. Defaults to `2` if
+     * unspecified. Use `"all"` to use all the availability zones in the current region.
      */
-    numberOfAvailabilityZones?: number | "all";
+    requestedAvailabilityZone?: string[] | "all";
```

Comment:
this would be a breaking change.  i think it would be fine to request specific AZs, but it should either be another value that can be passed to `numberOfAvailabilityZones`, or we would need a new property to pass this data in.